server/routes/api/choropleth.py

```diff
@@ -72,6 +72,8 @@ def get_choropleth_display_level(geoDcid):
     # Territories of the US, e.g. country/USA are also AA1. Restrict this view to only States.
     if geoDcid == 'country/USA':
         return geoDcid, 'State'
+    if geoDcid == 'Earth':
+        return geoDcid, 'Country'
```

Comment:
as more special handling of geoDcids gets added, I think it makes sense to create a map for these special geoDcids. The map would be similar to CHOROPLETH_DISPLAY_LEVEL_MAP but have key be geoDcid and value be display_level

Reply:
Done